.travis/matrix-installs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ev
+
+if [ "${MATRIX_TYPE}" = "python" ]; then
+    pip install coverage unittest2 mock --use-mirrors
+
+    # kafka install, copied from https://github.com/wvanbergen/kafka/blob/master/.travis.yml
+    wget http://www.us.apache.org/dist/kafka/0.8.2.1/kafka_2.10-0.8.2.1.tgz -O kafka.tgz
+    mkdir -p kafka && tar xzf kafka.tgz -C kafka --strip-components 1
+    nohup bash -c "cd kafka && bin/zookeeper-server-start.sh config/zookeeper.properties &"
+    nohup bash -c "cd kafka && bin/kafka-server-start.sh config/server.properties &"
+    sleep 5
+    kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic case --zookeeper localhost:2181
+elif [ "${MATRIX_TYPE}" = "javascript" ]; then
+    npm install -g bower
+    npm install -g grunt
+    npm install -g grunt-cli
+    ln -nfs `which bower` /home/travis/bower
+    python manage.py bower install
```

Comment:
(or alternative you can key off the test runner as well and just install it for that run)